### ISSUE TITLE
test.py is missing torch.no_grad()

### ISSUE THREAD

**Issue by Grabber** [dog-qiuqiu/Yolo-FastestV2#42]
For inference there is no need to calculate the gradients.

```
model.eval()
.
.
.
with torch.no_grad():
   preds = model(img)
```
